scripts/spark_wdl/ReadsPipelineSparkMulticore.wdl

```diff
@@ -0,0 +1,70 @@
+workflow ReadsPipelineSparkWorkflow {
+  File ref_fasta
+  File ref_fasta_index
+  File? ref_fasta_gzi
+  File ref_dict
+  File input_bam
+  String output_vcf_basename
+  File known_sites
+  File known_sites_index
+
+  String gatk_docker
+  String gatk
+  Int? num_cpu_override
+  Int num_cpu = select_first([num_cpu_override, 16])
+
+  call ReadsPipelineSpark {
+    input:
+      ref_fasta = ref_fasta,
+      ref_fasta_index = ref_fasta_index,
+      ref_fasta_gzi = ref_fasta_gzi,
+      ref_dict = ref_dict,
+      input_bam = input_bam,
+      output_vcf_basename = output_vcf_basename,
+      known_sites = known_sites,
+      known_sites_index = known_sites_index,
+      gatk_docker = gatk_docker,
+      gatk = gatk,
+      num_cpu = num_cpu
+  }
+
+  output {
+    File output_vcf = ReadsPipelineSpark.output_vcf
+  }
+}
+
+task ReadsPipelineSpark {
+  File ref_fasta
+  File ref_fasta_index
+  File? ref_fasta_gzi
+  File ref_dict
+  File input_bam
+  String output_vcf_basename
+  File known_sites
+  File known_sites_index
+
+  String gatk_docker
+  String gatk
+  Int num_cpu
+
+  command {
+    ${gatk} \
+      ReadsPipelineSpark \
```

Comment:
For locally run spark jobs I have found it prudent to explicitly specify the temp folder as something like `./tmp` to make sure its not accidentally trying to write that into the boot disk (which is usually vanishingly small)

Reply:
This was OK on Terra as it uses a sensible temp directory I think. I've now explicitly set the Spark temp directory for safety's sake.